tests/test_playbooks/content_view_version_cleanup_role.yml

```diff
@@ -70,8 +70,8 @@
     - name: check remaining content view versions
       assert:
         that:
-          - remaining_cvs.resources[0].versions|map(attribute='version')|list == ['3.0', '4.0', '5.0']
-          - remaining_cvs.resources[1].versions|map(attribute='version')|list == ['3.0', '4.0', '5.0']
+          - remaining_cvs.resources[0].versions|map(attribute='version')|list | difference(['3.0', '4.0', '5.0'])|length == 0
+          - remaining_cvs.resources[1].versions|map(attribute='version')|list | difference(['3.0', '4.0', '5.0'])|length == 0
```

Comment:
Those two are not equal. When the `remaining_cvs` list is a *subset* of 3,4,5, you still get `true`:

```console
% ansible -m debug -a "msg='{{ [1] | difference([1,2,3]) }}'" localhost
localhost | SUCCESS => {
    "msg": []
}
```

Reply:
In https://github.com/theforeman/foreman-ansible-modules/blob/1fe33bfd214cba3515f78a8706e7da6b5790bdf3/roles/content_view_version_cleanup/tasks/find_and_delete_unused_cv_versions.yml#L19 we use `map('float') | sort` to make things stable. Should we do the same here? Given we only have .0 versions, we can even avoid the floating conversion.

Reply:
https://github.com/theforeman/foreman-ansible-modules/pull/1758